src/main/java/frc/robot/Cal.java

```diff
@@ -69,30 +73,35 @@ public static final class Intake {
     public static final double INTAKING_POWER = 1.0;
     public static final double EJECTION_POWER = -1.0;
 
-    /** Input degrees, output [0,1] */
+    /** Input deg/s, output [0,1] */
     public static final double DEPLOY_MOTOR_P = Cal.PLACEHOLDER_DOUBLE,
         DEPLOY_MOTOR_I = Cal.PLACEHOLDER_DOUBLE,
         DEPLOY_MOTOR_D = Cal.PLACEHOLDER_DOUBLE;
 
     /** Intake positions in degrees */
-    public static final double STARTING_POSITION_DEGREES = 40.0,
-        DEPLOYED_POSITION_DEGREES = Cal.PLACEHOLDER_DOUBLE;
+    public static final double STARTING_POSITION_DEGREES = 38.0, DEPLOYED_POSITION_DEGREES = 180.0;
 
     /** Past this position, the intake is free to clamp */
-    public static final double CLAMP_POSITION_THRESHOLD_DEGREES = Cal.PLACEHOLDER_DOUBLE;
+    public static final double CLAMP_POSITION_THRESHOLD_DEGREES = 110.0;
 
     /** Absolute encoder position when the arm is at 0 degrees */
     public static final double ABSOLUTE_ENCODER_START_POS_DEG = Cal.PLACEHOLDER_DOUBLE;
 
     /** Voltage required to hold the intake in the horizontal position */
-    public static final double ARBITRARY_FEED_FORWARD_VOLTS = Cal.PLACEHOLDER_DOUBLE;
+    // Stall torque: 3.36 Nm * 75 = 252 Nm
+    // Max torque: 6 kg * 9.81 (N/kg) * 0.26m = 15.3 Nm
+    // Ratio: 0.73 V
```

Comment:
Should these be /** */ comments? (Will there be any reason to want to see this from somewhere else, or is it just for reference should we need it?)

E: applies also to similar comments, like lines 98-99

Reply:
nah they're just for reference, and I checked that the /**/ comments are still the hover-text